client/build/utils/createCanonicalLink.js

```diff
@@ -0,0 +1,55 @@
+"use strict";
+
+Object.defineProperty(exports, "__esModule", {
+  value: true
+});
+exports.default = void 0;
+
+var _siteConfig = _interopRequireDefault(require("@config/siteConfig.json"));
+
+function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
+
+var host = _siteConfig.default.details.host;
+
+var createBasicCanonicalLink = function createBasicCanonicalLink(page) {
+  return "".concat(host, "/").concat(page);
+};
+
+var createAssetCanonicalLink = function createAssetCanonicalLink(asset) {
+  var channelName, certificateId, name, claimId;
+
+  if (asset.claimData) {
+    var _asset$claimData = asset.claimData;
+    channelName = _asset$claimData.channelName;
+    certificateId = _asset$claimData.certificateId;
+    name = _asset$claimData.name;
+    claimId = _asset$claimData.claimId;
+  }
+
+  if (channelName) {
+    return "".concat(host, "/").concat(channelName, ":").concat(certificateId, "/").concat(name);
```

Comment:
use template strings?
e.g.
```
return `${host}/${channelName}:${certificateId}/${name}`
```

Reply:
This is the babel transpiled code :)

Reply:
once I merge the www.spee.ch and spee.ch repos we should be able to keep this out of version control